quinn-proto/src/connection/ack_frequency.rs

```diff
@@ -33,9 +33,18 @@ impl AckFrequencyState {
 
     /// Returns the `max_ack_delay` that should be requested of the peer when sending an
     /// ACK_FREQUENCY frame
-    pub(super) fn candidate_max_ack_delay(&self, config: &AckFrequencyConfig) -> Duration {
+    pub(super) fn candidate_max_ack_delay(
+        &self,
+        config: &AckFrequencyConfig,
+        peer_params: &TransportParameters,
+    ) -> Duration {
         // Use the peer's max_ack_delay if no custom max_ack_delay was provided in the config
-        config.max_ack_delay.unwrap_or(self.peer_max_ack_delay)
+        config
```

Comment:
We're requesting that the peer apply a certain upper bound, and the peer's `min_ack_delay` is the lowest value it will accept for that parameter. Our own `min_ack_delay` isn't involved, though the application can set `config.max_ack_delay` low if they want to prioritize fast loss detection.